docs/source/usage/plugins/radiation.rst

```diff
@@ -394,6 +398,12 @@ Dataset  Description                                           Dimensions
 ``z_Im`` imaginary part, z-component of the complex amplitude  (``N_observer``, ``N_omega``, 1)
 ======== ===================================================== ====================================
 
+.. note::
+
+   Please be aware, that despite the fact, that the SI-unit of each amplitude entry is :math:`\mathrm{[\sqrt{Js}]}`, the stored ``unitSI`` attribute returns :math:`\mathrm{[Js]}`.
+   This inconsistency will be fixed in the future.
```

Comment:
When reading this I have no idea what to do as a user.
Should I multiply with `sqrt(unitSI)` to get the right unit? I am lost.

Reply:
The statement is that the units of the _stored_ quantity **are** `[Js]` but **should actually be** `[\sqrt{Js}]` I would say that this is a hint to the user to take a square root of the _stored_ quantity. 

It is perhaps better to explicitly state that the user should perform the operation himself until the inconsistency is resolved.

Reply:
@ax3l I will find a more suitable wording tomorrow. @Anton-Le's explanation is correct.

Reply:
Just state what a user must do with the values and the attribute in order to get the right result. Then note the inconsistency.

Reply:
@PrometheusPi could you please fix this that we can merge it?